flex/interactive/examples/modern_graph/modern_graph_string_edge.yaml

```diff
@@ -0,0 +1,71 @@
+name: modern_graph # then must have a modern dir under ${data} directory
```

Comment:
I suggest moving this testing schema.yaml to `flex/test/rt_mutable_graph`, which is the place we put testing related stuffs.